setup.py

```diff
@@ -1,13 +1,8 @@
 from setuptools import setup, find_packages
 
-# Read in the version number
-__version__ = None  # This makes linters happy
-with open('conference_scheduler/version.py', 'r') as f:
-    exec(f.read())
-
 setup(
     name='conference-scheduler',
-    version=__version__,
+    version='1.1.0',
```

Comment:
The  idea is that `bumpversion` will rewrite this line for us? (I'm happy either way just making sure it's supposed to be in both places).